tools/appveyor/run.sh

```diff
@@ -6,20 +6,15 @@ CWD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd $CWD/../..
 ROOT=$PWD
 
-/c/Python37-x64/python -m pip install --upgrade pip
-
-cat appveyor/pip/numpy/numpy-1.16.4+mkl-cp37* > appveyor/pip/numpy-1.16.4+mkl-cp37-cp37m-win_amd64.whl
-/c/Python37-x64/Scripts/pip install appveyor/pip/numpy-1.16.4+mkl-cp37-cp37m-win_amd64.whl
-/c/Python37-x64/Scripts/pip install appveyor/pip/numpydoc-0.8.0-py2.py3-none-any.whl
-/c/Python37-x64/Scripts/pip install appveyor/pip/scipy-1.3.0-cp37-cp37m-win_amd64.whl
-/c/Python37-x64/Scripts/pip install -r requirements.txt
-/c/Python37-x64/Scripts/pip install tensorflow==1.14.0 psutil
+/c/Python36-x64/python -m pip install --upgrade pip
+/c/Python36-x64/Scripts/pip install -r requirements.txt
+/c/Python36-x64/Scripts/pip install psutil wheel
 
 set +e
-/c/Python37-x64/python setup.py build_ext -j 2 --inplace
+/c/Python36-x64/python setup.py build_ext -j 2 --inplace
```

Comment:
@Dekken just curious why is the change from Python 3.7 to 3.6 required?

Reply:
I'm testing appveyor to see what is causing the segfault

Reply:
so far it seems python3.6 is ok but python3.7 is not